City-of-Helsinki/drupal-helfi-kasvatus-koulutus
# UHF-4455 & UHF-3454 High school search map embed

# High school search map embed [UHF-4455](https://helsinkisolutionoffice.atlassian.net/browse/UHF-4455) & [UHF-3454](https://helsinkisolutionoffice.atlassian.net/browse/UHF-3454)
In addition to list view of high school search results, we added a map view of the same results.

## What was done
* A new iframe including the results on map was added.
* The list and map results were separated to two tabs and a javascript code was added to make it possible for the user to alternate between the two views.

## How to install
* Checkout this branch on your KASKO instance.
* `composer require drupal/hdbt:dev-UHF-4455_high_school_search_map_embed`
* `make drush-cim && make drush-cr`

## How to test
* If you have synced the database from testing you should have a page with both high school search and normal unit search: https://helfi-kasko.docker.so/fi/kasvatus-ja-koulutus/ammatillinen-koulutus/tutustu-ja-hae-ammatilliseen-koulutukseen
* On the page there on the high school search there should be "Show results on a map" tab on the right side of the results. Click on that and you should see the filtered results on a map. There should also be a link from where you can open the map in a new tab.
* Test the functionality and that the tab you have selected remains open after you adjust the filters.

## Other PRs
* https://github.com/City-of-Helsinki/drupal-hdbt/pull/259
